examples/nextjs-swc/src/pages/index.tsx

```diff
@@ -22,7 +22,7 @@ const Index: NextPage = () => {
    * This hook is needed to subscribe your
    * component for changes if you use t`` macro
    */
-  useLingui()
+  const i18n = useLingui()
```

Comment:
The comment above is outdated.

Also, wondering why in the previous example we used `t`, but here we use `i18n._(msg)`